README.md

```diff
@@ -239,8 +193,20 @@ All output from the console is synchronized to the file in real-time.
 
 If you close the program and then reopen it, a new log file will be created.
 
+
 ## Run the test
```

Comment:
...or should this section be another sub-section of `Development`?

Reply:
Add Development section and make logs, deploy and test into its sub-section